packages/schema/src/grid/observer.ts

```
@@ -0,0 +1,93 @@
+
+  addObserver(element: HTMLElement) {
+    const child = this.childList.find((t) => t.element === element);
+    if (!child) {
+      const childIndex = this.childList.length;
+      const child = {
+        element,
+        observer: new MutationObserver(this.callback),
+        dispose: () => {
+          if (child.observer) {
+            child.observer.disconnect();
+            delete child.observer;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)